pom.xml

```diff
@@ -23,8 +23,8 @@ under the License.
   <parent>
     <groupId>org.apache.maven</groupId>
     <artifactId>maven-parent</artifactId>
-    <version>38</version>
-    <relativePath/>
+    <version>39</version>
+    <relativePath />
```

Comment:
Please also change in `maven-bom` - on master I added comment here to remember about it